src/com/sun/ts/tests/jpa/core/annotations/mapkeytemporal/Department4.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2008, 2020 Oracle and/or its affiliates. All rights reserved.
  *
```

Comment:
This file doesn't contain any changes, so Copyright date change and change at end of file, should be reverted.

Reply:
Fixed.